### DISCUSSION TITLE
Foundations Course: Add WSL as an option for the installations section

### DISCUSSION THREAD

**Discussion by brandhawa99** [TheOdinProject/curriculum#25707] [Ideas]
### Describe your suggestion

Right now VM is the recommended option for Windows users getting started with The Odin Project. WSL (windows subsystem linux) is a great alternative for people  to use and can make the setup much easier.

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-installations

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

**Comment by CouchofTomato** [comment]
Thanks @brandhawa99 

I've converted this to a discussion as it's not an issue we can easily resolve in a short period of time.

**Comment by KevinMulhern** [comment]
This is in the works - https://github.com/TheOdinProject/curriculum/pull/25402

**Comment by Justin-Bento** [comment]
Wouldn't it be better if you demonstrated the process of setting up WSL2? Then, you could link to the rest of the setup for the Linux path. WSL2 allows you to run a Linux distribution alongside your Windows installation, providing a solid Linux kernel experience.

I've completed this process on my system, and everything is running smoothly to this day. This is fantastic since I don't have to run a separate operating system and worry about bricking my system, as my hardware doesn't play well with Linux.
